docs/CHANGELOG.md

```diff
@@ -24,19 +50,21 @@
 - Add harness metadata section ([#158](https://github.com/formatc1702/WireViz/issues/158), [#214](https://github.com/formatc1702/WireViz/pull/214))
 - Add support for supplier and supplier part number information ([#240](https://github.com/formatc1702/WireViz/issues/240), [#241](https://github.com/formatc1702/WireViz/pull/241/))
 - Add graph rendering options (colors, font, color name display style, ...) ([#158](https://github.com/formatc1702/WireViz/issues/158), [#214](https://github.com/formatc1702/WireViz/pull/214))
```

Comment:
These two bullet points seem to be duplicated just below. Normally, I would suggest deleting the latter copy, but I've already suggested a minor change to the latter copy in my first review 3 days ago, so I suggest deleting this copy instead:
```suggestion
```